src/style.css

```
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body,
+a,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
+  background-color: white;
+  margin: auto;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 50px;
+}
+
+.logo {
+  background-image: url("./images/logo.png");
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+}
+
+.nav__item {
+  display: inline-block;
+  position: relative;
+  margin: 0 0 0 20px;
+}
+
+.nav__item:first-child {
+  margin: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  --height: 60px;
+  height: var(--height);
```

you do not need variable here
```suggestion
  height: 60px;
```

---

changed